<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8" rowstretch="1,1,1,1" columnstretch="1,1,1" rowminimumheight="1,1,1,1" columnminimumwidth="1,1,1">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <property name="verticalSpacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>NP Retrosynthesis Screening</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_5_ShowRoute" stretch="0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string> Synthetic Pathway Visualization</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>325</width>
           <height>596</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_1_Input">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string> Target Molecule</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="chem_tab_inp">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <attribute name="title">
          <string>Input</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="3">
           <widget class="QPlainTextEdit" name="plainTextEdit_chem_inp"/>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_chem_upload">
            <property name="text">
             <string>Upload</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_chem_draw">
            <property name="text">
             <string>Draw</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_chem_add">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>List</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0" colspan="6">
           <widget class="QListWidget" name="listWidget_chem_list"/>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_chem_show">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_chem_predict">
            <property name="text">
             <string>Predict</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" colspan="3">
           <widget class="QPushButton" name="pushButton_chem_clear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_save">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_setting">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Setting</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_4_EC">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string> EC Number Prediction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_EC"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3_CompoundsList">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string> Compounds in Selected Pathway</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_CompoundsList">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2_RouteList">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string> Candidate Pathways</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_RouteList"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
